bin/boot

```diff
@@ -77,18 +81,29 @@ function start {
         fi
 
         echo $log_src[`date +%F.%H:%M:%S`]' Downloading additional addons...'
-        sudo -i -u \#$TARGET_ID python /opt/odoo/auto_addons/addons.py $FETCH_OCA_DEPENDENCIES $ADDONS_REPO
+        sudo -i -u $odoo_user python /opt/odoo/auto_addons/addons.py $FETCH_OCA_DEPENDENCIES $ADDONS_REPO
     else
-        echo $log_src[`date +%F.%H:%M:%S`]' No additional addons to download'
-        sudo -i -u \#$TARGET_ID bash /opt/odoo/auto_addons/no_addons.sh $odoo_conf_file
+        # No additional addons to download
+        sudo -i -u $odoo_user bash /opt/odoo/auto_addons/no_addons.sh $odoo_conf_file
     fi
+}
+
+function start {
+    echo $log_src[`date +%F.%H:%M:%S`]' Checking special requirements...'
+    bash /opt/scripts/startup.sh
+
+    echo $log_src[`date +%F.%H:%M:%S`]' Updating Odoo conf...'
+    _update_odoo_conf_params
+
+    echo $log_src[`date +%F.%H:%M:%S`]' Check if need to download addons repo...'
+    _download_addons
 
     echo $log_src[`date +%F.%H:%M:%S`]' Running odoo...'
     set +e
     if [ ! -e $1 ]; then
-        echo $log_src[`date +%F.%H:%M:%S`]' ...with additional args: $*'
+        echo $log_src[`date +%F.%H:%M:%S`]' ...with additional args: ' $*
     fi
-    sudo -i -u \#$TARGET_ID /usr/bin/python \
+    sudo -i -u $odoo_user /usr/bin/python \
```

Comment:
It would be dangerous to allow to change the path of the binary to be executed with a parameter